status/discover.go

```diff
@@ -39,147 +37,6 @@ func getClient() (kubernetes.IstioClientInterface, error) {
 	return client, nil
 }
 
```

Comment:
Is there no discovery anymore?

Reply:
No, at least for Jaeger, we are going to force the user set the endpoint